recipes/intel-ipsec-mb/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+from conans import tools, ConanFile, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class IntelIpSecMbConan(ConanFile):
+    name = "intel-ipsec-mb"
+    description = "Intel(R) Multi-Buffer Crypto for IPSec"
+    license = "BSD 3-Clause"
```

Comment:
Please always use SPDX Identifier

```suggestion
    license = "BSD-3-Clause"
```

https://spdx.org/licenses/